server/models/admin.sql

```diff
@@ -63,6 +63,11 @@ BEGIN
     SELECT voucher.uuid, CONCAT_WS('.', 'VO', project.abbr, voucher.reference)
     FROM voucher JOIN project where project.id = voucher.project_id;
 
+  -- stock_requisition
+  INSERT INTO document_map
+    SELECT stock_requisition.uuid, CONCAT_WS('.', 'SREQ', stock_requisition.reference)
```

Comment:
Normally, our document maps are composed of three different elements, but here you only use two.  Can you add in the project to make it into three?

Ex) Invoices -> `IV.HBB.123`
Ex) Stock Movements -> `SM.12.1234`